src/main/java/edu/ucsb/cs156/happiercows/entities/Commons.java

```diff
@@ -28,6 +27,7 @@ public class Commons {
     private double milkPrice;
     private double startingBalance;
     private LocalDateTime startingDate;
+    private LocalDateTime lastDate; // subject to change, see issue #18, linked to #11
```

Comment:
this comment won't make any sense in the final production repo, so let's remove this and make it as a comment on the PR instead.